commands/projects_test.go

```diff
@@ -96,14 +96,6 @@ func TestProjectsUpdateAllAttributes(t *testing.T) {
 		}
 		tm.projects.On("Update", projectUUID, updateReq).Return(&testProject, nil)
 
-		config.Doit.(*TestConfig).IsSetMap = map[string]bool{
```

Comment:
How do these removals relate to adding support for custom domains in Spaces CDN?

Reply:
I updated `TestConfig.Set` to add entries in `IsSetMap` automatically so these are redundant now. One of the tests in this file had arguments supplied via `Set` out of sync with mutation of `IsSetMap` here: https://github.com/digitalocean/doctl/pull/415/files/789465cc23fd6b4c7c4f1683f985e55cb8d3cd3b#diff-0f9818a0fd8799f01ac7bd4c2ba02fedL148

Reply:
Gotcha. Thanks for the explanation.